2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -244,7 +244,16 @@ function setup_keyfile() {
     exit 0
```

Comment:
Not only here, looking at `common.sh` there are many occurrences of `exit`... feels puzzling.